lib/utils.js

```diff
@@ -29,4 +29,28 @@ exports.basicAuth = function(username, password) {
 
 		next();
 	};
+}
+
+// Wrap require in a check for the module being present
+function requireNodeModule (modulePath) {
+	try {
+		nodeModule = require(modulePath);
+	}
+	catch(err) {
+		if (err.code !== 'MODULE_NOT_FOUND') {
+			// Re-throw not "Module not found" errors 
+			throw err;
+		}
+		else {
+			// Module not found error
+			console.error('\nERROR: ' + modulePath + ' module missing. Try running `npm install`\n')
+			process.exit(0);
+		}
+	}
+	return nodeModule;
+}
+
+module.exports = {
+	requireNodeModule: requireNodeModule,
+	basicAuth: basicAuth
 };
```

Comment:
This file doesn't follow our [styleguide for js](https://github.com/alphagov/styleguides/blob/master/js.md#whitespace) – soft tabs with two space indents